DESCRIPTION

```diff
@@ -21,8 +21,9 @@ Imports:
     tidyr (>= 0.6.0),
     lazyeval (>= 0.2.0),
     modelr (>= 0.1.0)
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
 Suggests: testthat,
     knitr,
-    rmarkdown
+    rmarkdown,
```

Comment:
Is there any new functionality that depends on resamplr? If not, please remove it from this list.